src/context.rs

```diff
@@ -60,8 +60,8 @@ pub mod global {
 }
 
 
-/// A trait for all kinds of Context's that lets you define the exact flags and a function to deallocate memory.
-/// It shouldn't be possible to implement this for types outside this crate.
+/// A trait for all kinds of contexts that lets you define the exact flags and a function to
+/// deallocate memory. It shouldn't be possible to implement this for types outside this crate.
```

Comment:
In 269bde042f9d6bfccb857c6921e71f01a9854f35:

You could change "shouldn't be possible" to "isn't possible". I think we've come to trust rustc more than when this comment was written :)